robottelo/ui/discoveryrules.py

```diff
@@ -28,17 +30,31 @@ def _configure_discovery(self, hostname=None, host_limit=None,
                 priority
             )
         self.assign_value(locators['discoveryrules.enabled'], enabled)
+        if locations:
+            self.configure_entity(
+                locations,
+                FILTER['discovery_rule_loc'],
+                tab_locator=tab_locators['tab_loc'],
+            )
+        if organizations:
+            self.configure_entity(
+                organizations,
+                FILTER['discovery_rule_org'],
+                tab_locator=tab_locators['tab_org'],
+            )
 
     def create(self, name, search_rule, hostgroup, hostname=None,
-               host_limit=None, priority=None, enabled=True):
+               host_limit=None, priority=None, locations=None,
+               organizations=None, select=True, enabled=True):
```

Comment:
Can't see any usage of `select` arg. Is it redundant or implementation is missing?